girder_worker/app.py

```diff
@@ -4,6 +4,7 @@
 from distutils.version import LooseVersion
 from celery.signals import (task_prerun, task_postrun,
                             task_failure, task_success, worker_ready)
+from girder_client import GirderClient
```

Comment:
If that's the case, we should move it from girder_io requirements into the main setup.py.